.github/workflows/release.yml

```diff
@@ -30,23 +26,21 @@ jobs:
 
       - name: NPM – Install Dependencies
         run: npm install
-        working-directory: ${{ github.workspace }}/PipelinesTasks/${{ matrix.task }}/src/task
 
       - name: NPM – Build
         run: npm run build:release
-        working-directory: ${{ github.workspace }}/PipelinesTasks/${{ matrix.task }}/src/task
 
       - name: Install TFX CLI
         run: npm install -g tfx-cli
 
       - name: Release – Create
-        run: tfx extension create --manifest-globs vss-extension.json --output-path ../ms-omex.${{ matrix.task }}.vsix
-        working-directory: ${{ github.workspace }}/PipelinesTasks/${{ matrix.task }}/build/release
+        run: tfx extension create --manifest-globs vss-extension.json --output-path ../ms-omex.PRMetrics}.vsix
```

Comment:
I think this should be (remove closing bracket):
run: tfx extension create --manifest-globs vss-extension.json --output-path ../**ms-omex.PRMetrics.vsix**